routes/reminders.js

```diff
@@ -11,8 +11,22 @@ function removeFamilyProperty(item) {
 }
 
 router.get('/', (req, res, next) => {
-  reminders.index(req.user.family, req.query.start, req.query.limit)
-  .then(rows => {
+  const start = parseInt(req.query.start);
+  let limit = parseInt(req.query.limit);
```

Comment:
`typeof NaN` returns 'number', doesn't it? In what cases are the InvalidInputError thrown?